src/WingetCreateTests/WingetCreateTests/UnitTests/UpdateCommandTests.cs

```diff
@@ -1,79 +1,106 @@
+        public void Setup()
+        {
+            this.sw = new StringWriter();
+            Console.SetOut(this.sw);
+            Logger.Initialize();
+        }
+
+        /// <summary>
+        /// Teardown method for the update command unit tests.
+        /// </summary>
+        [OneTimeTearDown]
+        public void TearDown()
+        {
+            this.sw.Dispose();
+        }
+
+        /// <summary>
+        /// Verifies that the update command modifies the manifest's version field
+        /// when a "Version" argument is provided.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Test]
+        public async Task UpdateVersion()
+        {
+            string version = "1.2.3.4";
+            UpdateCommand command = this.GetUpdateCommand(TestConstants.TestPackageIdentifier, version, this.tempPath);
+            Assert.IsTrue(await command.Execute(), "Command should have succeeded");
+
+            string manifestDir = Utils.GetAppManifestDirPath(TestConstants.TestPackageIdentifier, version);
+            string fullOutputPath = Path.Combine(this.tempPath, manifestDir, $"{TestConstants.TestPackageIdentifier}.yaml");
+
+            Assert.IsTrue(File.Exists(fullOutputPath), "Updated manifest was not created successfully");
+            string result = File.ReadAllText(fullOutputPath);
+            Assert.That(result, Does.Contain("PackageVersion: 1.2.3.4"), $"Failed to update version of {TestConstants.TestPackageIdentifier}");
+        }
+
+        /// <summary>
+        /// Tests exception handling for the GitHub repo manifest lookup functionality.
+        /// Passes an invalid package identifier. Expected to throw an Octokit Not Found Exception.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Test]
+        public async Task UpdateWithInvalidPackageIdentifier()
+        {
+            string tempPath = Path.GetTempPath();
+            UpdateCommand command = this.GetUpdateCommand(TestConstants.TestInvalidPackageIdentifier, null, this.tempPath);
+            Assert.IsFalse(await command.Execute(), "Command should have failed");
+            string result = this.sw.ToString();
+            Assert.That(result, Does.Contain(Resources.OctokitNotFound_Error), "Octokit.NotFoundException should be thrown");
+        }
+
+        /// <summary>
+        /// Tests the update command to ensure that updating a multiple installer manifests should throw an error.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Test]
+        public async Task UpdateWithMultipleInstallers()
+        {
+            UpdateCommand command = this.GetUpdateCommand(TestConstants.TestMultipleInstallerPackageIdentifier, null, this.tempPath);
+            Assert.IsFalse(await command.Execute(), "Command should have failed");

```

Comment:
>Execute [](http://example.com/codeflow?start=41&length=7)

Isn't the point of refactoring UpdateCommand.cs so that you can pass manifest yamls directly to it from test cases, instead of requiring new files in winget-pkgs-submission-test?

Reply:
You are right, I wasn't quite sure if we wanted to completely abandon the entire update command/github flow, but I see your point now. I have made the change so that we can pass local manifests directly to the tests